install/terminal/select-dev-language.sh

```diff
@@ -6,6 +6,16 @@ else
 	languages=$(gum choose "${AVAILABLE_LANGUAGES[@]}" --no-limit --height 10 --header "Select programming languages")
 fi
 
+enable_lazyvim_extras() {
+	local temp_file=$(mktemp /tmp/omakub.XXXXX)
+	local extras=("$@")
+	local jq_extras=$(printf '"%s",' "${extras[@]}")
+	jq_extras="[${jq_extras%,}]"
+
+	jq --argjson extras "$jq_extras" '.extras |= (. + $extras | unique)' ~/.config/nvim/lazyvim.json >"$temp_file" &&
+	mv "$temp_file" ~/.config/nvim/lazyvim.json
```

Comment:
Rather than the temp file dance, I would just touch ~/.config/nvim/lazyvim.json first, then assume it's there.

Reply:
We can't read and write to the same file in a pipeline like that, something to do with it using the same file descriptor. I tried here, and the config file ended up empty.

However, I found some tricks to achieve this behavior, as described [here](https://backreference.org/2011/01/29/in-place-editing-of-files/). I've updated the code to use one of the tricks described there and it looks like it works.

Reply:
(happy to revert it to use a tmp file, if you don't want to use this trick, btw)